content/includes/veteran-navigation.html

```diff
@@ -224,6 +224,7 @@
                   <li><a href="/burials-and-memorials/" onClick="reportHeaderNav('Explore Benefits->Burials Memorials');">Burials &amp; memorials overview</a></li>
                   <li><a href="/burials-and-memorials/eligibility/" onClick="reportHeaderNav('Explore Benefits->Burials Memorials');">Eligibility</a></li>
                   <li><a href="/burials-and-memorials/burial-planning/" onClick="reportHeaderNav('Explore Benefits->Burials Memorials');">Plan a burial</a></li>
+                  <li><a href="/burials-and-memorials/pre-need/" onClick="reportHeaderNav('Explore Benefits->Burials Memorials');">Plan ahead for your own burial by <br>applying for pre-need eligibility</a></li>
```

Comment:
That you need to put a `<br>` in here is a bug, I think. Can you open a ticket about text not wrapping in the navigation menu?